webapp/client/public/css/base.css

```diff
@@ -126,6 +126,11 @@ input.form-control.field-error-found {
   min-height: 55px;
 }
 
+.steuerlotse-select:focus {
+  border: 2px solid var(--focus-border-color);
+  box-shadow: 0 0 0 2px var(--focus-color);
+}
+
```

Comment:
Why does this also need to go into the base.css?

Reply:
Because so far the focus of that component was missing and this should add this not only for the react component but the old component as well.